src/main/scala/io/opentargets/etl/backend/spark/Helpers.scala

```diff
@@ -141,19 +141,27 @@ object Helpers extends LazyLogging {
             format: String,
             header: Option[String],
             delimiter: Option[Boolean]
-          ) if format.contains("sv") => {
+          ) if format.contains("sv") =>
```

Comment:
@JarrodBaker do we want here `.endsWith("sv")` more than `contains`?

Reply:
That would be safer, but I thought I'd seen an input file somewhere which was compressed (`csv.gz`) and wanted to capture that as well. We discussed the other week that isn't a good situation as it can't be partitioned. I'll change it to `endsWith` as a way to enforcing that compressed csv's aren't given as inputs.

Reply:
@JarrodBaker makes sense, we can leave it not allowing the gz and when we face a case where partioned files are comrpessed then we can rethink this again.